tools/travis-test.sh

```diff
@@ -223,7 +223,7 @@ elif [ -n "$USE_SDIST" ] && [ $# -eq 0 ]; then
   $PYTHON -c "import fcntl; fcntl.fcntl(1, fcntl.F_SETFL, 0)"
   # ensure some warnings are not issued
   export CFLAGS=$CFLAGS" -Wno-sign-compare -Wno-unused-result -Wno-error=undef"
-  $PYTHON setup.py sdist
+  $PYTHON -m build --sdist
```

Comment:
Is `build` installed? The [CI fails](https://github.com/numpy/numpy/actions/runs/5813680535/job/15761994512?pr=24379#step:4:333) with
```
+ python -m build --sdist
/home/runner/work/numpy/numpy/builds/venv/bin/python: No module named build
```